babybuddy/tests/formats/tests_en_gb.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+import datetime
+
+from django.core.exceptions import ValidationError
+from django.forms.fields import DateTimeField
+from django.test import TestCase, override_settings  # , tag
+from django.utils.formats import date_format, time_format
+
+from babybuddy.middleware import update_en_gb_date_formats
+
+
+class GbFormatsTestCase(TestCase):
+    @override_settings(LANGUAGE_CODE='en-GB')
+    def test_datetime_input_formats(self):
+        update_en_gb_date_formats()
+        field = DateTimeField()
+        supported_custom_examples = [
+            '20/01/2020',
+            '20/01/2020 9:30 AM',
+            '20/01/2020 9:30:03 AM',
+            '01/10/2020 11:30 PM',
+            '01/10/2020 11:30:03 AM',
+        ]
+
+        for example in supported_custom_examples:
+            try:
+                result = field.to_python(example)
+                self.assertIsInstance(result, datetime.datetime)
+            except ValidationError:
+                self.fail('Format of "{}" not recognized!'.format(example))
+
+        with self.assertRaises(ValidationError):
+            field.to_python('invalid date string!')
+
+    # @tag('isolate')
```

Comment:
@Amith211 I think we don't need to isolate these tests anymore. So you should be able to safely remove this tag and import.